pyravendb/store/session_query.py

```diff
@@ -1019,7 +1019,7 @@ def _execute_query(self):
         response_includes = response.pop("Includes", None)
         self.session.save_includes(response_includes)
         for result in response_results:
-            (entity, metadata, original_metadata, original_document,) = Utils.convert_to_entity(
+            (entity, metadata, original_document,) = Utils.convert_to_entity(
```

Comment:
what's that last left `,` for? How is it diff from https://github.com/ravendb/ravendb-python-client/pull/95/files#diff-cdc2009938de871a418ec044668392680f1374f02281db86f3251d8b38dc1c09R431

Reply:
In this case, it's the same, but a comma at the end of the collection reduces diffs and including it (even inline) is considered good practice.
https://stackoverflow.com/a/7992643